docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -11,47 +11,65 @@ GloWGR: Whole Genome Regression
     covariates_csv = 'test-data/gwas/covariates.csv.gz'
     continuous_phenotypes_csv = 'test-data/gwas/continuous-phenotypes.csv.gz'
 
-Glow supports Whole Genome Regression (WGR) as GloWGR, a parallelized version of the
-`regenie <https://rgcgithub.github.io/regenie/>`_ method (see the
-`preprint <https://www.biorxiv.org/content/10.1101/2020.06.19.162354v2>`_).
+Glow supports Whole Genome Regression (WGR) as GloWGR, a parallelized version of the `regenie <https://rgcgithub.github.io/regenie/>`_ method (see the `preprint <https://www.biorxiv.org/content/10.1101/2020.06.19.162354v2>`_). GloWGR supports two types of phenotypes:
+
+- Quantitative 
+- Binary
+
+Many steps of the GloWGR workflow explained in this page are common between the two cases. Any step that is different between the two has  separate explanations clearly marked by "for quantitative phenotypes" vs. "for binary phenotypes".
+
+-----------
+Performance
+-----------
+
+The following figures demonstrates the performance gain obtained by using parallelized GloWGR in comparision with single machine BOLT, fastGWA GRM, and regenie for quantitative and binary phenotypes.
+
+.. TODO: We need to add more information on the experiments behind this graphs.
 
 .. image:: ../_static/images/wgr_runtime.png
    :scale: 50 %
 
+.. TODO: Add a picture for binary
+
+--------
+Overview
+--------
+
 GloWGR consists of the following stages.
 
-- Block the genotype matrix across samples and variants.
-- Perform dimensionality reduction with ridge regression.
-- Estimate phenotypic values with ridge regression.
+- Block the genotype matrix across samples and variants
+- Perform dimensionality reduction with linear ridge regression
+- Estimate phenotypic predictors using
 
-.. note::
+  - **For quantitative phenotypes**: linear ridge regression
+  - **For binary phenotypes**: logistic ridge regression
+
+The following diagram provides an overview of the operations and data within the main stages of GlowWGR and their interrelationship.
+
+.. image:: ../_static/images/wgr_diagram.png
```

Comment:
In the diagram, I don't think there should be an arrow from the prediction probabilities to Glow GWAS. You might use predicted probabilities to fill in missing phenotype values, but not as covariates in an association scan.

Reply:
Changed following the change of API.